packages/entity-server/src/app/createApp.ts

```diff
@@ -33,11 +37,16 @@ export function createApp() {
     .useBasicBearerAuth('entity-server')
     .use<SingleEntityRequest | MultiEntityRequest>('hierarchy/:hierarchyName', attachCommonContext)
     .use<MultiEntityRequest>('hierarchy/:hierarchyName/descendants', attachMultiEntityContext)
+    .use<MultiEntityRequest>('hierarchy/:hierarchyName/relatives', attachMultiEntityContext)
```

Comment:
Now we have `/relatives` and `/relations`. Is this too confusing? 😂 

Initially I definitely felt that way, and was trying to decide which to rename. However, as I worked on this more I did come to feel they are fairly conceptually linked. Relations are the relations among the relatives, and relatives are just the full set of all relations (within a hierarchy).

But there's a good chance I've just gotten too used to the code on this one, and if at first glance it's confusing I am definitely happy to change it (might do it regardless).

I'm a tad reluctant to rename `/relations` as it's really a direct resource for our `entityRelations` table, I think that naming consistency is nice. So it comes to picking a new name for `/relatives`?

Reply:
It really is confusing, and I'm keen for a rename

I get where you're coming from, but am not totally convinced that `/relations` needs to remain...would you consider `/relationships`, which seems more descriptive to me and makes the distinction between that and `/relatives` pretty clear (at least in my mind)?

Reply:
As per our discussion, gonna rename `/relations` to `/relationships`! Think this should be a pretty safe change as I think only the Aggregator uses this endpoint for now